tests/HUBViewControllerTests.m

```diff
@@ -2866,6 +2874,68 @@ - (void)testThatDelegateIsNotifiedWhenHeaderAppears
     XCTAssertEqualObjects(self.componentViewsFromApperanceDelegateMethod, @[self.component.view]);
 }
 
+- (void)testThatModelIsSetImmediatelyOnInitialRenderRequest
+{
+    HUBViewModelRendererMock *viewModelRenderer = [HUBViewModelRendererMock new];
+    [self createViewControllerWithViewModelRenderer:viewModelRenderer];
+
+    self.contentOperation.contentLoadingBlock = ^(id<HUBViewModelBuilder> viewModelBuilder) {
+        [viewModelBuilder headerComponentModelBuilder].title = @"title1";
+        return YES;
+    };
+
+    [self.viewController.view layoutIfNeeded];
+    [self.viewController reload];
```

Comment:
Instead of this, how about calling `[self simulateViewControllerLayoutCycle]`? (to be consistent with the other tests)